src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -898,6 +953,13 @@ QString EnzymeTreeItem::getEnzymeInfo() const {
     QString result;
     result += QString("<a href=\"http://rebase.neb.com/rebase/enz/%1.html\">%1</a>")
                   .arg(text(Column::Id));
+    if (enzymesNumber != INCORRECT_ENZYMES_NUMBER) {
+        if (enzymesNumber > MAXIMUM_ENZYMES_NUMBER) {
+            result += tr(" (>10000 sites)");
```

Comment:
I do not see this text in russian.ts. Looks like tr() does not work here as expected.

Reply:
Just forgot to update translation file, fixed